spec/index.bs

```diff
@@ -10845,6 +10843,72 @@ It must only be included by interfaces which also include those mixins.
         </div>
 </dl>
 
+<div class=note heading id=dynamicOffsetOrder>
+    Dynamic offset are applied in {{GPUBindGroupLayoutEntry}}.{{GPUBindGroupLayoutEntry/binding}} order.
+
+    This means that if `dynamic bindings` is the list of each {{GPUBindGroupLayoutEntry}} in the {{GPUBindGroupLayout}}
+    with {{GPUBindGroupLayoutEntry/buffer}}?.{{GPUBufferBindingLayout/hasDynamicOffset}} set to `true`, sorted by
+    {{GPUBindGroupLayoutEntry}}.{{GPUBindGroupLayoutEntry/binding}}, then `dynamic offset[i]`, as supplied to
+    [=GPUBindingCommandsMixin/setBindGroup()=], will correspond to `dynamic bindings[i]`.
+
+    <div class=example>
+        For a {{GPUBindGroupLayout}} created with the following call:
+
+        <pre highlight=js>
+            // Note the order of the binding properties!
+            let layout = gpuDevice.createBindGroupLayout({
+                entries: [{
+                    binding: 1,
+                    buffer: {},
+                }, {
+                    binding: 2,
+                    buffer: { dynamicOffset: true },
+                }, {
+                    binding: 0,
+                    buffer: { dynamicOffset: true },
+                }]
+            });
+        </pre>
+
+        Used by a {{GPUBindGroup}} created with the following call:
+
+        <pre highlight=js>
+            // Note the order of the binding properties!
+            let bindGroup = gpuDevice.createBindGroup({
+                layout: layout,
+                entries: [{
+                    binding: 1,
+                    resource: { buffer: bufferA },
```

Comment:
Maybe since we're going through a whole example already, it would be useful to show how static offsets apply too? (the static and dynamic offsets add together)

Reply:
Done!